### PR TITLE
CLOUDP-98165: Allow mongocli to create gov only projects

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-98165

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli iam project create --help
Create a project.

Usage:
  mongocli iam projects create <name> [flags]

Flags:
      --govCloudRegionsOnly   Make the project gov region only.
  -h, --help                  help for create
      --orgId string          Organization ID to use. Overrides configuration file or environment variable settings.
  -o, --output string         Output format.
                              Valid values: json|json-path|go-template|go-template-file
      --ownerId string        Unique 24-hexadecimal digit string that identifies the Atlas user account to be granted the Project Owner role on the specified project.

Global Flags:
  -P, --profile string   Profile to use from your configuration file.
```

```json
./bin/mongocli iam project create test1 --govCloudRegionsOnly -P cloud-dev-gov -o json
{
  "id": "6141ffb3998ba22ac3476d39",
  "orgId": "60e409d39e80a53e4f4163a7",
  "name": "test1",
  "created": "2021-09-15T14:14:14Z",
  "links": [
    {
      "rel": "self",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39"
    },
    {
      "rel": "http://cloud.mongodb.com/containers",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/containers"
    },
    {
      "rel": "http://cloud.mongodb.com/clusters",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/clusters"
    },
    {
      "rel": "http://cloud.mongodb.com/databaseUsers",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/databaseUsers"
    },
    {
      "rel": "http://cloud.mongodb.com/peers",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/peers"
    },
    {
      "rel": "http://cloud.mongodb.com/processes",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/processes"
    },
    {
      "rel": "http://cloud.mongodb.com/whitelist",
      "href": "https://cloud-dev.mongodbgov.com/api/atlas/v1.0/groups/6141ffb3998ba22ac3476d39/whitelist"
    }
  ]
}
```
Screenshot 2021-09-15 at 15 15 34

### CODE REVIEW

File: internal/usage/usage.go [hunk 1]
```diff
@@ -264,6 +264,7 @@ const (
 	ContainerRegion                 = "Atlas region where the container resides."
 	ContainerRegions                = "List of Atlas regions where the container resides."
 	ProjectOwnerID                  = "Unique 24-hexadecimal digit string that identifies the Atlas user account to be granted the Project Owner role on the specified project."
+	GovCloudRegionsOnly             = "Make the project gov region only."
```
Comment (gssbzn) [inline internal/usage/usage.go:267]: this should probably clarify only available for Atlas Gov

**Review by gssbzn — DISMISSED** [review]
LGTM pending docs review

File: internal/usage/usage.go (hunk 1 above)
Comment (melissamahoney-mongodb) [reply internal/usage/usage.go:267]: Added more detail. Is this correct: if you use this flag the project uses GovCloud region, and if you don't use it then the project uses Standard region?
```suggestion
	GovCloudRegionsOnly             = "Only for AtlasGov organizations. If specified, designates that the project uses the AWS GovCloud region only. If unspecified, the project uses the AWS Standard region only. You can't deploy clusters across AWS GovCloud and AWS Standard regions in the same project."
```

**Review by melissamahoney-mongodb — DISMISSED** [review]

**Comment by gssbzn** [comment]
Conflict

**Comment by andreaangiolillo** [comment]
@gssbzn ready now

**Review by fmenezes — APPROVED** [review]
restamp

**Review by melissamahoney-mongodb — APPROVED** [review]

**Comment by andreaangiolillo** [comment]
@melissamahoney-mongodb I need a restamp

**Pull request merged**